frontend/utils/branding.tsx

```diff
@@ -252,6 +252,7 @@ const sites = {
 }
 
 export const TICKETING_PAYMENT_ENABLED = false
+export const QUEUE_ENABLED = false
```

Comment:
Is it possible to pull this from the backend settings? Not a huge deal if not.

Reply:
Couldn't find a way from first glance. Could potentially use env variables maybe instead of separate settings, but would be more elegant in the long term to implement some sort of global variables table so admin etc. can also toggle queue + other features with a gui.